src/test/java/org/mifos/processor/cucumber/stepdef/ConfigurationTestStepDef.java

```diff
@@ -0,0 +1,89 @@
+package org.mifos.processor.cucumber.stepdef;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.camel.Exchange;
+import org.mifos.processor.bulk.config.PaymentModeMapping;
+import org.mifos.processor.bulk.config.PaymentModeType;
+import org.mifos.processor.bulk.utility.Utils;
+
+import java.util.function.Function;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public class ConfigurationTestStepDef extends BaseStepDef {
+
+    @Given("Application context is loaded")
+    public void applicationContextIsLoaded() {
+        System.out.println("Context is loaded");
+    }
+
+    @When("I assert the payment mode config")
+    public void paymentModeConfigAssert() {
+        assertThat(paymentModeConfiguration).isNotNull();
+    }
+
+    @Then("I should get the non empty payment modes")
+    public void nonEmptyPaymentModesCheck() {
+        assertThat(paymentModeConfiguration.getMappings()).isNotEmpty();
+        for (PaymentModeMapping mapping : paymentModeConfiguration.getMappings()) {
+            System.out.println(mapping.getId());
+            System.out.println(mapping.getType());
+            System.out.println(mapping.getEndpoint());
+        }
+    }
+
+    @And("I should be able fetch the mapping for mode {string}")
+    public void fetchMappingForMode(String mode) {
+        PaymentModeMapping mapping = paymentModeConfiguration.getByMode(mode);
+        assertThat(mapping).isNotNull();
+        BaseStepDef.paymentModeMapping = mapping;
```

Comment:
we need to make sure that all the tests are working on a particular tenant. so that all these static/class-level values are unique.

Reply:
this is kind of unit tests. So how tenant is related??